try {
    data.push(value);
    alert('Action completed!');
    console.log('Processing data...');
} catch (error) {
    console.error("Error:", error);
}